opal/util/Makefile.am

```diff
@@ -138,18 +142,12 @@ libopalutil_core_la_DEPENDENCIES = \
         json/libopalutil_json.la \
         keyval/libopalutilkeyval.la
 
-# flex prior to version 2.6.6 uses the POSIX extension fileno()
-# without providing the proper feature test macro, so
-# we add the _POSIX_C_SOURCE macro here.
-# See https://github.com/westes/flex/issues/263
-libopalutil_core_la_CFLAGS = -D_POSIX_C_SOURCE=200809L
-
 # Conditionally install the header files
 
 if WANT_INSTALL_HEADERS
 opaldir = $(opalincludedir)/$(subdir)
 opal_HEADERS = $(headers)
 endif
 
-maintainer-clean-local:
-	rm -f show_help_lex.c
+# This file is generated
+CLEANFILES = show_help_content.c
```

Comment:
This should be in `maintainer-clean-local`.  Otherwise, it shouldn't be in `EXTRA_DIST`, as "make clean" would remove artifacts that were in the source tarball.  I think `make distcheck` only works here because the remove actually fails, the source tree being chmod'ed to read-only during the test.